<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomePage</class>
 <widget class="QWidget" name="WelcomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Coursework</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>340</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>13</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Welcome to my Graphics Engine. Please select one of the following options:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="renderButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Quick Render</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>450</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>By Tanaz Siriwardena</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
